cloudflare/data_cloudflare_devices.go

```diff
@@ -0,0 +1,73 @@
+package cloudflare
```

Comment:
this file should be renamed to match the remainder of the data sources `data_cloudflare_devices.go` => `data_source_cloudflare_devices.go`